src/operations/booleans/index.js

```diff
@@ -0,0 +1,5 @@
+module.exports = {
+  intersect: require('./intersect'),
```

Comment:
hmm the naming contradicts our existing API : difference vs subtract, intersect vs intersection, we should be coherent here

Reply:
difference (NOUN)
subtract (VERB)

intersection (NOUN)
intersect (VERB)

union (NOUN)
?? (VERB)

By our standards, we should be using VERBS. So, 'subtract' and 'intersect' seem correct.

However, we are current using 'union' which is a NOUN. Not good. Would could change to 'combine' or 'merge' or something else.

Reply:
When Union is used to reference the mathematical operation such as our use case, it is a verb. See https://en.m.wiktionary.org/wiki/union#Verb

Reply:
@RedHatter thanks for the support!

Reply:
I think we have this correct. Let's leave them as VERBS.

@kaosat-dev Are you fine with this?

Reply:
sure, let's go with that then